nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service-api/src/main/java/org/apache/nifi/elasticsearch/IndexOperationResponse.java

```diff
@@ -40,7 +40,8 @@ public boolean hasErrors() {
         return hasErrors;
     }
 
-    public static IndexOperationResponse fromJsonResponse(String response) throws IOException {
+    @SuppressWarnings("unchecked")
+    public static IndexOperationResponse fromJsonResponse(final String response) throws IOException {
         ObjectMapper mapper = new ObjectMapper();
```

Comment:
Use of an `ObjectReader` would require an `ObjectMapper` to be created first (the `ObjectReader` constructors need either an `ObjectMapper` or `ObjectReader`)... but absolutely, this should be a class-level constant, so that's easy to do